bin/node/Cargo.toml

```diff
@@ -20,6 +20,7 @@ tracing-forest = ["miden-node-block-producer/tracing-forest"]
 [dependencies]
 anyhow                    = { version = "1.0" }
 clap                      = { version = "4.5", features = ["derive", "env", "string"] }
+humantime                 = { version = "2.2" }
```

Comment:
I'm unsure if its worth adding this. It lets you use `10h 5m 3s`. The alternative is just keeping normal `Duration` and documenting that its an integer in milliseconds.

Reply:
Not a strong opinion at all but given the choice I'd keep milliseconds as a fixed unit instead

Reply:
Yes, I'd probably just keep it as milliseconds.